### ISSUE TITLE
Scaling the unit covariance gaussian to the PCA space

### ISSUE THREAD

**Issue by chahuja** [HackerPoet/FaceEditor#10]
As eigen values of the covariance matrix (`1/n * X^T @ X`) are the square of the eigen values of the random variable `X`, the scaling factor should be `e**0.5` instead of `e`.   
https://github.com/HackerPoet/FaceEditor/blob/69519606315925c237485a82223391cb3f511ec3/encoder.py#L144

**Comment by martianmartina** [comment]
Yes, one singular value of X should be the square root of an eigenvalue of the covariance matrix of X. I think the problem is fixed in face_edit.py where he took the square root after loading evals.npy, but I agree that this could cause a problem in encoder.py. Good catch!
